src/react-image-lightbox.js

```diff
@@ -1497,6 +1497,7 @@ class ReactImageLightbox extends Component {
               className="ril-prev-button ril__navButtons ril__navButtonPrev"
               key="prev"
               aria-label={this.props.prevLabel}
+              title={this.props.nextLabel}
```

Comment:
`this.props.prevLabel` instead of `this.props.nextLabel`

Reply:
You're right, thank you